frontend/src/components/AboutModal.jsx

```diff
@@ -0,0 +1,53 @@
+import React from 'react';
+import Modal from '@material-ui/core/Modal';
+import {
+  Box,
+  Typography,
+  Button,
+  IconButton,
+} from '@material-ui/core';
+import CLOSE from '../images/svg/close.svg';
+
+const AboutModal = (props) => {
+  const { open, handleClose, title } = props;
+  return (
+    <Modal
+      open={open}
+      className="primary-modal"
+      onClose={handleClose}
+    >
+      <Box className="modal-dialog">
+        <Box className="modal-header">
+          <Typography>{`About ${title}`}</Typography>
+          <IconButton
+            color="inherit"
+            onClick={handleClose}
+            disableFocusRipple
+            disableRipple
+          >
+            <img src={CLOSE} alt="Close" />
+          </IconButton>
+        </Box>
+        <Box className="modal-body">
+          <Typography>
+            {`${title} is an initiative from the Yale University for Neurosciences, in partnership
```

Comment:
Would suggest to extract the modal content to variables in constants.js

The actual text will be provided in a later stage by the client